LayoutTests/fast/css/text-underline-offset-repaint-expected.txt

```diff
@@ -0,0 +1,6 @@
+Text
+
+(repaint rects
+  (rect 8 16 784 18)
+)
+
```

Comment:
I suggest to use fixed height font here (Ahem) to make sure repaint rects stay consistent across platforms.

Reply:
I've updated. Thanks!

Reply:
it looks like you are missing an extra line at the end.

Reply:
I rushed a bit and updated without checking locally. Sorry. It should work correctly this time.